python/src/tests/tasks/test_list_genes.py

```diff
@@ -8,88 +8,126 @@
 
 
 class TestListGenes:
-
     @pytest.fixture(autouse=True)
     def load_correct_definition(self):
-        self.correct_request_skeleton = {
+        self.correct_desc = {
+            "body": {
+                "name": "ListGenes",
+                "selectFields": ["gene_names", "dispersions"],
+                "orderBy": "dispersions",
+                "orderDirection": "DESC",
+                "offset": 0,
+                "limit": 20,
+            }
+        }
+        self.correct_asc = {
             "body": {
                 "name": "ListGenes",
                 "selectFields": ["gene_names", "dispersions"],
                 "orderBy": "dispersions",
-                "orderDirection": "desc",
+                "orderDirection": "ASC",
                 "offset": 0,
                 "limit": 20,
             }
         }
+        self.correct_names = {
+            "body": {
+                "name": "ListGenes",
+                "selectFields": ["gene_names", "dispersions"],
+                "orderBy": "gene_names",
+                "orderDirection": "DESC",
+                "offset": 0,
+                "limit": 20,
+            }
+        }
+        self.correct_filter = {
+            "body": {
+                "name": "ListGenes",
+                "selectFields": ["gene_names", "dispersions"],
+                "orderBy": "gene_names",
+                "orderDirection": "DESC",
+                "offset": 0,
+                "limit": 40,
+                "geneNamesFilter": "LIN",
+            }
+        }
+        self.correct_startswith = {
+            "body": {
+                "name": "ListGenes",
+                "selectFields": ["gene_names", "dispersions"],
+                "orderBy": "gene_names",
+                "orderDirection": "DESC",
+                "offset": 0,
+                "limit": 40,
+                "geneNamesFilter": "^LIN",
+            }
+        }
+        self.correct_empty = {
+            "body": {
+                "name": "ListGenes",
+                "selectFields": ["gene_names", "dispersions"],
+                "orderBy": "gene_names",
+                "orderDirection": "DESC",
+                "offset": 0,
+                "limit": 40,
+                "geneNamesFilter": "^Glefjskamsmesascc",
+            }
+        }
+        self.correct_response = json.load(open(os.path.join("tests", "lg_result.json")))
```

Comment:
```suggestion
        self.correct_response = json.load(open(os.path.join("tests", "list_genes_expected.json")))
```
Might make sense to use explicit file names that match tests (applies to other expected results as well).